plugin/build.gradle

```diff
@@ -240,7 +238,11 @@ List<String> jacocoExclusions = [
         'org.opensearch.ml.profile.MLModelProfile',
         'org.opensearch.ml.profile.MLPredictRequestStats',
         'org.opensearch.ml.action.load.TransportLoadModelAction',
+        'org.opensearch.ml.action.load.TransportLoadModelOnNodeAction',
         'org.opensearch.ml.model.MLModelManager',
+        'org.opensearch.ml.action.unload.TransportUnloadModelAction',
+        'org.opensearch.ml.action.forward.TransportForwardAction',
+        'org.opensearch.ml.action.syncup.TransportSyncUpOnNodeAction',
```

Comment:
Why adding these files to exclusions?

Reply:
these are  original content,

Reply:
you should rebase the main branch first to avoid the confused change shown in your PR.

Reply:
rollback